In-house patch to remove the imports of local copies of six and
ordereddict. This is Solaris-specific and not suitable for upstream
submission.

--- urllib3-1.15.1/urllib3/_collections.py.~1~	2016-04-06 12:16:56.000000000 -0700
+++ urllib3-1.15.1/urllib3/_collections.py	2016-06-21 16:53:35.000000000 -0700
@@ -14,8 +14,8 @@ except ImportError:  # Platform-specific
 try:  # Python 2.7+
     from collections import OrderedDict
 except ImportError:
-    from .packages.ordered_dict import OrderedDict
-from .packages.six import iterkeys, itervalues, PY3
+    from ordereddict import OrderedDict
+from six import iterkeys, itervalues, PY3
 
 
 __all__ = ['RecentlyUsedContainer', 'HTTPHeaderDict']
--- urllib3-1.15.1/urllib3/connection.py.~1~	2016-04-06 12:16:56.000000000 -0700
+++ urllib3-1.15.1/urllib3/connection.py	2016-06-21 16:53:51.000000000 -0700
@@ -6,7 +6,7 @@ import sys
 import socket
 from socket import error as SocketError, timeout as SocketTimeout
 import warnings
-from .packages import six
+import six
 
 try:  # Python 3
     from http.client import HTTPConnection as _HTTPConnection
--- urllib3-1.15.1/urllib3/connectionpool.py.~1~	2016-04-06 12:16:56.000000000 -0700
+++ urllib3-1.15.1/urllib3/connectionpool.py	2016-06-21 16:54:10.000000000 -0700
@@ -31,7 +31,7 @@ from .exceptions import (
     NewConnectionError,
 )
 from .packages.ssl_match_hostname import CertificateError
-from .packages import six
+import six
 from .connection import (
     port_by_scheme,
     DummyConnection,
--- urllib3-1.15.1/urllib3/contrib/appengine.py.~1~	2015-12-29 12:28:18.000000000 -0800
+++ urllib3-1.15.1/urllib3/contrib/appengine.py	2016-06-21 16:54:28.000000000 -0700
@@ -12,7 +12,7 @@ from ..exceptions import (
     SSLError
 )
 
-from ..packages.six import BytesIO
+from six import BytesIO
 from ..request import RequestMethods
 from ..response import HTTPResponse
 from ..util.timeout import Timeout
--- urllib3-1.15.1/urllib3/fields.py.~1~	2016-04-06 12:16:56.000000000 -0700
+++ urllib3-1.15.1/urllib3/fields.py	2016-06-21 16:54:54.000000000 -0700
@@ -2,7 +2,7 @@ from __future__ import absolute_import
 import email.utils
 import mimetypes
 
-from .packages import six
+import six
 
 
 def guess_content_type(filename, default='application/octet-stream'):
--- urllib3-1.15.1/urllib3/filepost.py.~1~	2015-12-14 13:06:26.000000000 -0800
+++ urllib3-1.15.1/urllib3/filepost.py	2016-06-21 16:55:27.000000000 -0700
@@ -4,8 +4,8 @@ import codecs
 from uuid import uuid4
 from io import BytesIO
 
-from .packages import six
-from .packages.six import b
+import six
+from six import b
 from .fields import RequestField
 
 writer = codecs.lookup('utf-8')[3]
--- urllib3-1.15.1/urllib3/response.py.~1~	2016-04-06 12:16:56.000000000 -0700
+++ urllib3-1.15.1/urllib3/response.py	2016-06-21 16:55:48.000000000 -0700
@@ -9,8 +9,8 @@ from ._collections import HTTPHeaderDict
 from .exceptions import (
     ProtocolError, DecodeError, ReadTimeoutError, ResponseNotChunked
 )
-from .packages.six import string_types as basestring, binary_type, PY3
-from .packages.six.moves import http_client as httplib
+from six import string_types as basestring, binary_type, PY3
+from six.moves import http_client as httplib
 from .connection import HTTPException, BaseSSLError
 from .util.response import is_fp_closed, is_response_to_head
 
--- urllib3-1.15.1/urllib3/util/request.py.~1~	2015-12-14 13:06:26.000000000 -0800
+++ urllib3-1.15.1/urllib3/util/request.py	2016-06-21 16:55:59.000000000 -0700
@@ -1,7 +1,7 @@
 from __future__ import absolute_import
 from base64 import b64encode
 
-from ..packages.six import b
+from six import b
 
 ACCEPT_ENCODING = 'gzip,deflate'
 
--- urllib3-1.15.1/urllib3/util/response.py.~1~	2015-12-29 12:28:18.000000000 -0800
+++ urllib3-1.15.1/urllib3/util/response.py	2016-06-21 16:56:58.000000000 -0700
@@ -1,5 +1,5 @@
 from __future__ import absolute_import
-from ..packages.six.moves import http_client as httplib
+from six.moves import http_client as httplib
 
 from ..exceptions import HeaderParsingError
 
--- urllib3-1.15.1/urllib3/util/retry.py.~1~	2016-04-06 12:16:56.000000000 -0700
+++ urllib3-1.15.1/urllib3/util/retry.py	2016-06-21 16:57:44.000000000 -0700
@@ -9,7 +9,7 @@ from ..exceptions import (
     ReadTimeoutError,
     ResponseError,
 )
-from ..packages import six
+import six
 
 
 log = logging.getLogger(__name__)
